docs/openapi/components/schemas/common/LEIaddress.yml

```
@@ -1,8 +1,8 @@
 $linkedData:
   term: LEIaddress
-  '@id': https://w3id.org/traceability#LEIaddress
+  '@id': https://www.gleif.org/ontology/Base/PhysicalAddress
 title: LEIaddress
-description: GLEIF LEI Redord Level 1 Schema for LEI Address
+description: GLEIF Level 1 Ontology for LEI Address: https://www.gleif.org/ontology/v1.0/Base/index-en.html#PhysicalAddress and https://www.gleif.org/about-lei/common-data-file-format/current-versions/level-1-data-lei-cdf-3-1-format/2021-03-04_lei-cdf-v3-1.xsd
```

prefer markdown for these

---

The `2021-03-04_lei-cdf-v3-1.xsd` document doesn't contain `physical` nor `Physical`. It seems to be an erroneous citation. Please correct, or leave out (as suggested here).
```suggestion
description: GLEIF Level 1 Ontology for LEI Address: [physical address](https://www.gleif.org/ontology/v1.0/Base/index-en.html#PhysicalAddress)
```

---

@TallTed I've cited from the official LEI ontology.

---

Added fix